packages/python/plotly/_plotly_utils/utils.py

```diff
@@ -7,9 +7,6 @@
 from _plotly_utils.basevalidators import ImageUriValidator
 
 
-PY36_OR_LATER = sys.version_info.major == 3 and sys.version_info.minor >= 6
```

Comment:
we should probably leave this and change it to the new pattern, in case someone had the bad idea of referring to it

Reply:
Good idea, updated in https://github.com/plotly/plotly.py/pull/2078/commits/0fd758010549b2f1cfbb51743bca3b7c2a2e0b78.